### PR TITLE
spectator final decision UI

### PR DESCRIPTION
**Pull request by AdityaHarvi**
If/when spectators make a final decision on what event they would like to cause, a message will be updated on screen for all users.

### CODE REVIEW

File: client/src/scene/SceneGameArena.ts [hunk 1]
```diff
@@ -83,13 +86,36 @@ export class SceneGameArena extends Phaser.Scene {
...
+            "current event"
+        );
+        this.spectatorDecision = this.add
+            .bitmapText(
+                BOARD_PX - 13.75 * TILE_SIZE,
+                (BOARD_PX - 11 * TILE_SIZE) * 45,
+                "brawl",
+                ""
+            )
+            .setVisible(false);
```
Comment (swebra) [inline client/src/scene/SceneGameArena.ts:103]: I don't see this being set true anywhere and I don't think we need it... I thought we'd always have something showing with the default text being something like "None"

**Review by swebra — CHANGES_REQUESTED** [review]
Just to keep SceneGameArena focused primarily on the main gameplay stuff I'd maybe abstract this out into its own UI class like the controlsUI? Though I get why you didn't because its so small. idk, use your best judgement.

**Comment by AdityaHarvi** [comment]
I'll leave it up to @swebra to finish off the positioning of the text with the new background img (or I can take it on as well, if you wanted to push that change first)

**Comment by swebra** [comment]
Closing this PR and its branch in favor of #87 which includes the changes.

**Pull request closed without merging**